Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -301,7 +300,7 @@ def packet_gpio_gps_standby_disable():
 
     .. todo:: This function needs updating to the new create_gpio_command_packet()
     """
-    return create_gpio_command_packet(3, gpio_allocation.GPS_STANDBY, 1)
+    return create_gpio_command_packet(3, gpioallocations.GPS_STANDBY, 1)
```

Comment:
@reillyeon is this you adding this or the automated tests? This was done in https://github.com/reillyeon/Faraday-Software/commit/6060044dedfafc2afb574df8bf3263e7e955c0bf where you fixed undefined variables. Is this considered one?